parachain/pallets/ethereum-beacon-client/src/lib.rs

```diff
@@ -290,14 +303,66 @@ pub mod pallet {
 				(update.next_sync_committee_update.is_some() &&
 					update_attested_period == store_period);
 			ensure!(
-				update.attested_header.slot > store_period || update_has_next_sync_committee,
```

Comment:
bug fix here. should be `update.attested_header.slot > latest_finalized_state.slot`

Reply:
Nice catch!